csharp-basics/exercises/Arithmetic/CozaLozaWoza/CozaLozaWozaTest/CozaLozaWozaTest.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+
+namespace CozaLozaWoza.Test
+{
+    [TestClass]
+    public class CozaLozaWozaTest
+    {
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_Number3_ExpectedStringCoza()
+        {
+            // Arrange
+            var number = 3;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
+            //Assert
+            Assert.AreEqual("Coza", result);
+        }
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_Number5_ExpectedStringLoza()
+        {
+            // Arrange
+            var number = 5;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
+            //Assert
+            Assert.AreEqual("Loza", result);
+        }
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_Number7_ExpectedStringWoza()
+        {
+            // Arrange
+            var number = 7;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
+            //Assert
+            Assert.AreEqual("Woza", result);
+        }
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_MultipleOf3And5_ExpectedStringCozaLoza()
+        {
+            // Arrange
+            var number = 15;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
+            //Assert
+            Assert.AreEqual("CozaLoza", result);
+        }
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_MultipleOf3And7_ExpectedStringCozaWoza()
+        {
+            // Arrange
+            var number = 21;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
+            //Assert
+            Assert.AreEqual("CozaWoza", result);
+        }
+
+        [TestMethod]
+        public void CheckNumberForCozaLozaWoza_MultipleOf5And7_ExpectedStringLozaWoza()
+        {
+            // Arrange
+            var number = 35;
+            //Act
+            var result = CozaLozaWoza.CheckNumber(number);
```

Comment:
Neizskatās, ka tev uzd ir pareizi izpildīs. Write a program called CozaLozaWoza which prints the numbers 1 to 110, 11 numbers per line. The program shall print "Coza" in place of the numbers which are multiples of 3, "Loza" for multiples of 5, "Woza" for multiples of 7, "CozaLoza" for multiples of 3 and 5, and so on. The output shall look like:

1 2 Coza 4 Loza Coza Woza 8 Coza Loza 11 
Coza 13 Woza CozaLoza 16 17 Coza 19 Loza CozaWoza 22 
23 Coza Loza 26 Coza Woza 29 CozaLoza 31 32 Coza

Reply:
Uzdevums izpildās, kā nosacījumos rakstīts. Es testēju tikai to daļu, kur pārbaudīja vai cipars atbilst kādam no CozaLozaWoza nosacījumiem. Pašu izvadi netestēju.